ivy/functional/frontends/jax/random.py

```diff
@@ -45,3 +45,9 @@ def _get_seed(key):
 def beta(key, a, b, shape=None, dtype=None):
     seed = _get_seed(key)
     return ivy.beta(a, b, shape=shape, dtype=dtype, seed=seed)
+
+
+def dirichlet(key, alpha, shape=None, dtype="float32"):
+    seed = _get_seed(key)
+    alpha = ivy.array(alpha, dtype=dtype)
```

Comment:
Explicitly converting `alpha` into an `ivy.array` won't be necessary since this will be handled automatically inside the `@to_ivy_arrays_and_back` decorator.